doc/pages/user/095-known_issues.md

```diff
@@ -0,0 +1,40 @@
+# Known issues
+
+This page collects some notable issues which affect Astarte `v1.0`. This is by no means an
+exhaustive list and you should also check [Github
+issues](https://github.com/astarte-platform/astarte/issues) to see if your problem is already
+covered there.
+
+## Realm deletion
+
+Astarte `v1.0` introduces the possibility of deleting a Realm, but currently devices which have a
+valid certificate are not disconnected from the Realm. The issue is tracked
+[here](https://github.com/astarte-platform/astarte/issues/443).
+
+Since publishing with devices on unexisting realms can cause problems (namely: RabbitMQ data queues
+filling up) which can also impact devices on other realms, a realm should be deleted only after
+ensuring that no devices are connected to it.
+
+Due to the problems that realm deletion can cause, currently the feature must be explicitly enabled
+with a feature gate, i.e. by adding
+
+```yaml
+  features:
+    realmDeletion: true
+  components:
+    ...
+```
+
+to the Astarte Custom Resource (which maps to setting the `HOUSEKEEPING_ENABLE_REALM_DELETION`
+environment variable to `true` in the `astarte_housekeeping` container).
+
+## Group permissions
+
+Currently Astarte [authorization mechanism](070-auth.html) doesn't allow permissions on groups with
+a device granularity. Specifically there's no way to authorize a user to add only specific devices
+to a group. The issue is tracked [here](https://github.com/astarte-platform/astarte/issues/463).
+
+This means that right now the best way to allow users to add or remove devices from a group they
+have access to is to provide a backend which is able to perform the necessary authorization checks
+and then performs the necessary additions/removals, while the user only has a read access to the
+group resource.
```

Comment:
On the long term a minor semantic change is going to employed, therefore we discourage using etc. etc..